packages/libabsolute/libabsolute.0.1/opam

```diff
@@ -23,9 +23,9 @@ depends: [
 synopsis: "Libabsolute"
 description: "Libabsolute is a constraint programming library based on abstract domains from the theory of abstract interpretation. It is used and distributed with the AbSolute constraint solver."
 url {
-  src: "https://github.com/mpelleau/AbSolute/archive/0.2.tar.gz"
+  src: "https://github.com/mpelleau/AbSolute/archive/0.3.tar.gz"
   checksum: [
-    "md5=7fa3ddb22997078b47f517ada5f6cb96"
-    "sha512=43aff8a0f05022b90ff89f0b2feacdc829e9e6da9be88f802254aff383823b2d8f3dc4e3a694a9d534493d3e60b8a98de9c45a4690c3275b58a6a9dba476e133"
+    "md5=0dcb117b36b1afc68ba2c57186f6d04e"
+    "sha512=efcb9bf2a7fb5c62a00b35e5427064af65208186be0a6d343ac56206d5466951ef4593db04dc412987a11018f75d2952d4ab0e2384f588befaa50c8e1921133f"
```

Comment:
I don't think you should change this file. What's the reason for it?

Reply:
Yes you're right, i think it was changed automatically by ````opam publish````

Reply:
makes sense!